warden/src/main/kotlin/AttestationService.kt

```diff
@@ -284,17 +291,38 @@ abstract class AttestationService {
                 }
             }
 
-            is AttestationResult.Error -> when (val secondTry =
-                kotlin.runCatching {
-                    verifyAttestation(attestationProof, expectedChallenge, (keyToBeAttested as ECPublicKey).toAnsi())
-                }.getOrElse { return KeyAttestation(null, firstTry) }) {
-                is AttestationResult.Android -> throw RuntimeException("WTF?")
-                is AttestationResult.Error -> KeyAttestation(null, firstTry)
-                is AttestationResult.IOS -> KeyAttestation(keyToBeAttested, secondTry)
+            is AttestationResult.Error -> {
+                //try different encodings
+                val publicKeyEncodings = CryptoPublicKey.fromJcaPublicKey(keyToBeAttested).getOrThrow().let {
+                    listOf(
+                        it.iosEncoded,
+                        (it as CryptoPublicKey.EC).copy(
```

Comment:
Indendation makes it really hard to read and grasp what's going on here

Reply:
split everything up